$NetBSD: patch-core_src_cats_make__catalog__backup.pl.in,v 1.2 2021/02/01 09:08:43 kardel Exp $

	include ${PREFIX}/{bin,sbin} in path to find db programs/tools

--- core/src/cats/make_catalog_backup.pl.in.orig	2020-07-09 17:39:20.000000000 +0000
+++ core/src/cats/make_catalog_backup.pl.in
@@ -36,6 +36,7 @@ use strict;
 my $cat = shift or die "Usage: $0 catalogname";
 my $dir_conf='@sbindir@/bareos-dbcheck -B -c @confdir@';
 my $wd = "@working_dir@";
+$ENV{PATH} = "@PREFIX@/bin:@PREFIX@/sbin:" . $ENV{PATH};
 
 sub dump_sqlite3
 {
